### DISCUSSION TITLE
resource_group_name

### DISCUSSION THREAD

**Discussion by skelso01** [globalbao/terraform-azurerm-policy-exemptions#8] [Q&A] [answered]
Good morning,
while searching for a terraform resource, which creates an exeption, I found [this](https://github.com/hashicorp/terraform-provider-azurerm/issues/9197) issue on the azurerm provider github repository. My question now is, for what do I need this resource group name? I just want to create an exemption. In the documentation of MS there is no resource group needed to create a policy exemption for a resource. So why do I need to add a resource group name here?

**Comment by jesseloudon** [comment, marked as answer]
> Good morning,
> 
> while searching for a terraform resource, which creates an exeption, I found [this](https://github.com/hashicorp/terraform-provider-azurerm/issues/9197) issue on the azurerm provider github repository. My question now is, for what do I need this resource group name? I just want to create an exemption. In the documentation of MS there is no resource group needed to create a policy exemption for a resource. So why do I need to add a resource group name here?

Hi there 

Great question!

The RG name is needed because this Terraform module uses an resource group deployment template to manage the policy exemption. There's documentation links to this effect on this module's readme.

I have #2 open & waiting for me to action which will allow an individual resource to be exempted from a policy assignment. Currently we can manage an policy exemption for an RG only.

Hope this helps explain
Jesse

**Reply by skelso01** [reply]
Thx for the quick answer.
